scripts/CustomMaterials.zs

```diff
@@ -419,30 +466,48 @@ function generateHighPressureGases(material as string, id as int){
     .components([Utils.material(material) * 1])
     .flags(["disable_decomposition"])
     .build();
+
+    MaterialBuilder(id + 3, "hot_" + material)
+    .fluid()
+    .fluidTemp(323)
+    .color(color)
+    .components([Utils.material(material) * 1])
+    .flags(["disable_decomposition"])
+    .build();
 }
 
 generateHighPressureGases("hydrogen", 10000);
 
-generateHighPressureGases("oxygen", 10003);
+generateHighPressureGases("oxygen", 10004);
 
-generateHighPressureGases("helium", 10006);
+generateHighPressureGases("helium", 10008);
 
-generateHighPressureGases("neon", 10009);
+generateHighPressureGases("neon", 10012);
 
-generateHighPressureGases("argon", 10012);
+generateHighPressureGases("argon", 10016);
 
-generateHighPressureGases("krypton", 10015);
+generateHighPressureGases("krypton", 10020);
 
-generateHighPressureGases("xenon", 10018);
+generateHighPressureGases("xenon", 10024);
 
-generateHighPressureGases("air", 10021);
+generateHighPressureGases("air", 10028);
 
-generateHighPressureGases("nitrogen", 10024);
+generateHighPressureGases("nitrogen", 10032);
 
-function generateLiquidFromGas(material as string, id as int, boilingTemperature as int){
+generateHighPressureGases("nether_air", 10036);
+
+generateHighPressureGases("ender_air", 10040);
```

Comment:
I wonder if doing fluids for these types of systems with contenttweaker instead of GT will be better here. It means that you have more IDs for solids where GT's system is more useful.